Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/ExposureWindowRiskLevelConfig.kt

```diff
@@ -0,0 +1,18 @@
+package de.rki.coronawarnapp.appconfig
+
+import de.rki.coronawarnapp.server.protocols.internal.v2.AppConfigAndroid
+import de.rki.coronawarnapp.server.protocols.internal.v2.RiskCalculationParametersOuterClass
+
+interface ExposureWindowRiskLevelConfig {
```

Comment:
It's not the ExposureWindowRisk<i>Level</i>Config,_ it's the ExposureWindowRisk<i>Calculation</i>Config

Reply:
True, will fix 👍